src/main/resources/graphql/game.graphqls

```diff
@@ -12,8 +12,38 @@ type Game {
     visitorTeam: Team!
 }
 
+input GameInput {
+    id: ID!,
```

Comment:
We can leave generating the id to the application, that will make the client's job easier

Reply:
Done! 🔥